src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -66,19 +68,15 @@ internal void LogDiagnostic(Diagnostic diagnostic, CultureInfo culture)
                 message = "<None>";
```

Comment:
Does this really happen? Is it helpful to emit "<None>" to SARIF if this value is empty? Probably better to leave it as empty (i.e., don't emit it to SARIF at all). EXCEPT: message is non-optional, i see, by spec. hm. @lgolding. either message needs to be an optional thing or we need to populate the message here with something helpful (could we use the Descriptor title perhaps)?